core/src/main/java/nu/yona/server/subscriptions/service/BuddyConnectResponseMessageDto.java

```diff
@@ -160,7 +158,7 @@ MessageActionDto handleAction_Process(UserDto actingUser, BuddyConnectResponseMe
 					.createInstanceActionDone(theDtoFactory.createInstance(actingUser, connectResponseMessageEntity));
 		}
 
-		private void logHandledAction_Process(UserDto actingUser, BuddyConnectResponseMessage connectResponseMessageEntity)
+		private void logHandledAction_Process(User actingUser, BuddyConnectResponseMessage connectResponseMessageEntity)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS00100)